crates/matrix-sdk-appservice/src/lib.rs

```diff
@@ -392,7 +389,7 @@ impl AppService {
     /// active virtual clients.
     ///
     /// [transaction]: https://spec.matrix.org/v1.2/application-service-api/#put_matrixappv1transactionstxnid
-    pub async fn receive_transaction(
+    async fn receive_transaction(
```

Comment:
Why?

Reply:
The PR message mentions this:
> Move tests into lib.rs so that we can make receive_transaction private

Reply:
🙊

Reply:
Yeah, not exposing the functionality makes sure that consumers aren't able to introduce sife-effects into the store.